app/models/manageiq/providers/container_manager.rb

```diff
@@ -30,5 +32,10 @@ def aggregate_cpu_total_cores(targets = nil)
     def aggregate_memory(targets = nil)
       aggregate_hardware(:computer_systems, :memory_mb, targets)
     end
+
+    def add_default_policies
+      # make sure that add_policy indeed adds a policy profile (aka MiqPolicySet)
+      add_policy(MiqPolicySet.find_by_guid('747a70d0-a8bf-4b63-bdad-1676bc0c5356'))
```

Comment:
@blomquisg @gtanzillo @jrafanie I got this comment from all of you at some point so I assume I don't understand how guids are used in manageiq.

While ids are database specific and names can be changed Isn't that exactly what uuids are for? I'm also using this at the [seed patch](#7413) to connect between related entities which might also be wrong (As @jrafanie suggested)



> @blomquisg Looks like that guid is contained within the seed data here 7558
> 
> But, I'm also uncomfortable with referencing it with the guid. Maybe you can give it a real name instead using the guid and then do a find_by_name instead. That would make this code much easier to read.

Sure. Is it alright to seed a tag connected to that policy and then connect all tagged policies? Are tags for policies supported? (I'm trying to avoid the db change)